src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -189,4 +190,58 @@ public static Deadline parseDeadline(String deadline) throws ParseException {
         }
         return new Deadline(trimmedDeadline);
     }
+
+    /**
+     * Returns the day given in a string representing a date
+     */
+    public static int getDay(String date) {
+        // defensive programming
+        Matcher datetimeMatcher = Deadline.PATTERN_DATE.matcher(date);
+
+        if (datetimeMatcher.find()) {
+            return Integer.parseInt(datetimeMatcher.group(1));
+        } else {
+            return 0; // will eventually throw an error
+        }
+    }
+
+    /**
+     * Returns the month given in a string representing a date
+     */
+    public static int getMonth(String date) {
+        // defensive programming
+        Matcher datetimeMatcher = Deadline.PATTERN_DATE.matcher(date);
+
+        if (datetimeMatcher.find()) {
+            return Integer.parseInt(datetimeMatcher.group(2));
+        } else {
+            return 0; // will eventually throw an error
```

Comment:
Same for here

Reply:
For day and month, 0 is considered invalid day/month so `LocalDate::of` will just throw a `DateTimeException` when trying to create the `LocalDate` object

Reply:
Thanks for clarifying!